examples/post_training_quantization/openvino/llm_compression/README.md

```diff
@@ -0,0 +1,26 @@
+# Large Language Models Weight Compression Example
```

Comment:
I think it's worth to split Post-Training Quantization and Weights Compression in different folders just like we have it in NNCF documentation.

Reply:
I don't mind. The intent the layout you mentioned in OpenVINO docs is just to make it better visible for newcomers because formally LLM compression is a part of PTQ.